salt/roster/__init__.py

```diff
@@ -104,6 +104,7 @@ def targets(self, tgt, tgt_type):
             except salt.exceptions.SaltRenderError as exc:
                 log.error('Unable to render roster file: {0}'.format(exc))
             except IOError as exc:
+                log.error('Can\'t access roster for backend {0}: {1}'.format(back, exc))
```

Comment:
It is @thatch45 that hates double-quotes. :-) But I think he will agree that `"Fred's bar"` is better than `'Fred\'s bar`. And we still have plenty of [similar cases](https://github.com/saltstack/salt/blob/develop/salt/transport/raet.py#L96).